pkgs/development/python-modules/recordlinkage/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, jellyfish, numpy, pandas, scipy, scikit-learn, joblib
+, networkx, bottleneck, numexpr, pyarrow, pytest, fetchPypi }:
+buildPythonPackage rec {
+  pname = "recordlinkage";
+  version = "0.14";
+
```

Only [Python > 3.5](https://github.com/J535D165/recordlinkage/blob/master/setup.py#L48) is supported. I suggest to even go a step further and set it to 3.7 as everything older is EOL. 

```suggestion
  version = "0.14";

  disabled = pythonOlder "3.7";
```